homeassistant/components/myq/cover.py

```diff
@@ -43,14 +44,14 @@ def __init__(self, coordinator, device):
         """Initialize with API object, device id."""
         super().__init__(coordinator)
         self._device = device
-
-    @property
-    def device_class(self):
-        """Define this cover as a garage door."""
-        device_type = self._device.device_type
-        if device_type is not None and device_type == MYQ_DEVICE_TYPE_GATE:
-            return DEVICE_CLASS_GATE
-        return DEVICE_CLASS_GARAGE
+        if (
+            device.device_type is not None
+            and device.device_type == MYQ_DEVICE_TYPE_GATE
+        ):
```

Comment:
```suggestion
        if device.device_type == MYQ_DEVICE_TYPE_GATE:
```

Do we need the `None` check?

Reply:
I was just looking at that. I had it copied from original but I don't think we do. Will remove it.